src/Sentry/Sentry.csproj

```diff
@@ -16,8 +16,14 @@
 
   <PropertyGroup Condition="'$(SolutionName)' == 'Sentry.Unity'">
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <!-- To be able to make the internals visible, since we're not signing the assembly for Unity -->
+    <SignAssembly>false</SignAssembly>
```

Comment:
Does this affect things like assembly aliasing? Since before we did have signing and now we wouldn't?

Reply:
I did some extensive checking and we're good!